### PR TITLE
Introduce + element

### PR DESCRIPTION
**Pull request by namtn-2711**
Introduce + element

### CODE REVIEW

**Review by tuanlvt-1117 — APPROVED** [review]

**Review by vietnh-1109 — APPROVED** [review]

**Review by tuannm-1276 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  8. [Xử lí đồng thời](#xử-lí-đồng-thời)
+  9. [Xử lí lỗi](#xử-lí-lỗi)
+  10. [Định dạng](#định-dạng)
+  11. [Viết chú thích](#viết-chú-thích)
+  12. [Các ngôn ngữ khác](#các-ngôn-ngữ-khác)
+
+
+## Giới thiệu
+![Humorous image of software quality estimation as a count of how many expletives
+you shout when reading code](http://www.osnews.com/images/comics/wtfm.jpg)
```
Comment (daolq-2712) [inline README.md:19]:
```suggestion
you shout when reading code](http://www.osnews.com/images/comics/wtfm.jpg)

```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+```swift
+var address = "One Infinite Loop, Cupertino 95014"
+var city = address.split(separator: ",")[0]
+var zipCode = address.split(separator: ",")[1]
+saveCityZipCode(city , zipCode)
+```
+**[⬆ Về đầu trang](#mục-lục)**
+
+### Tránh hại não người khác
+Tường minh thì tốt hơn là ẩn.
```
Comment (daolq-2712) [inline README.md:111]:
```suggestion
Tường minh thì tốt hơn là ẩn.

```

**Review by daolq-2712 — CHANGES_REQUESTED** [review]
Pls check my comment!

**Review by daolq-2712 — APPROVED** [review]
LGTM!

**Pull request merged**